koku/koku/celery.py

```diff
@@ -139,6 +139,13 @@ def task(self, *args, **kwargs):
     }
 
 
+# Beat used to crawl the account hierarchy
+app.conf.beat_schedule["crawl_account_hierarchy"] = {
+    "task": "masu.celery.tasks.crawl_account_hierarchy",
+    "schedule": crontab(hour=0, minute=0),
```

Comment:
Does it make sense to make this configurable with an ENV?

Reply:
I think that we originally started down that route but after discussing with @adberglund we decided that this was definitely a daily task and would not make sense for it to be changed. The only value that I could see an ENV adding would be for local testing. But I'll loop @kholdaway in here too since he was part of that decision.

Reply:
I was thinking through the ordering of org updates and cost usage report ingestion and when the user would actually see the data.

Did you consider adding the work that crawl_account_hierarchy does to be part of the regular aws processing flow?  This is going to be difficult to add automated tests for.  Putting it in the processing flow would also give the user a refreshed view of the organization. 

If it must stay as a beat schedule. Maybe add the task to the /download admin endpoint.

Reply:
We can talk through that tomorrow. I am not sure it’s related to that flow at all. Ordering doesn’t matter. We take a snapshot of the tree daily. It can change before or after that snapshot. So controlling order doesn’t seem worth coupling it together. 

For test we created a tool to inject a tree into the database. I would say we leave it as is and not add a flag with low value in production.

Reply:
Sounds great.  Admittedly, I have very limited knowledge of AWS org units, so that would be helpful!

Reply:
Also, we touched base with Luke and he said he wouldn't be testing with real AWS accounts.  I forgot to mention that detail too.  I am not opposed to tweaking it if needed.  As Ashley said, when we chatted with Andrew he said let's add it when we know why we need it.